.pre-commit-config.yaml

```diff
@@ -36,6 +36,7 @@ repos:
         "types": [python]
         files: *files
         require_serial: true
+        exclude: ^(tests/fixtures)
```

Comment:
why was this added?

Reply:
We will discuss that when we talk about the tests